dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -0,0 +1,343 @@
+import { css } from '@emotion/react';
+import { getCookie, setCookie } from '@guardian/libs';
+import {
+	body,
+	from,
+	headline,
+	palette,
+	space,
+} from '@guardian/source-foundations';
+import { Button, Radio, RadioGroup } from '@guardian/source-react-components';
+import { useState } from 'react';
+import type { EditionId } from '../lib/edition';
+import { getEditionFromId } from '../lib/edition';
+import { guard } from '../lib/guard';
+import { useOnce } from '../lib/useOnce';
+import { SvgClose } from './SvgClose';
```

Comment:
I think there's a built in Source SVG that we can use for this instead of creating our own!

https://guardian.github.io/csnx/?path=/docs/source-react-components_icons--docs